cfme/cloud/tenant.py

```diff
@@ -430,3 +439,29 @@ def step(self, *args, **kwargs):
             self.prerequisite_view.toolbar.configuration.item_select('Edit Cloud Tenant')
         else:
             raise DestinationNotFound('Cannot edit Cloud Tenants in CFME < 5.7')
+
+
+@navigator.register(Tenant, 'CloudSubnets')
+class CloudSubnets(CFMENavigateStep):
+    VIEW = ParentWithSubnetView
+    prerequisite = NavigateToSibling('Details')
+
+    def step(self):
+        item = 'Cloud Subnets'
+        if not int(self.prerequisite_view.entities.relationships.get_text_of(item)):
+            raise DestinationNotFound("This Cloud Tenant doesn't have {item}".format(item=item))
```

Comment:
I'm thinking this message (in both nav destinations) should include the tenant object as well, for reference.

Something like:
`raise DestinationNotFound('Cloud Tenant {} has a 0 count for {} relationships'.format(self.obj, item)`